_includes/activity_pub/image_modal.html

```diff
@@ -0,0 +1,27 @@
+{%- comment -%}
```

Comment:
Is this being used?

Reply:
No, I removed it
ea11f2b76a8e4338f34370146e2dcf7984c8bf7f